examples/common/utilities.adb

```diff
@@ -27,6 +34,46 @@ package body Utilities is
 
     --  ------------------------------------------------------------------------
 
+    procedure Print_GL_Array3 (Name : String; anArray : GL.Types.Single_Array) is
```

Comment:
OK, I'll check this out.
I may have misunderstood what GL.Types.Singles.Vector3_Array is.
Certainly would be better if I can use it.